ipypublish/preprocessors/slides_from_markdown.py

```diff
@@ -174,8 +174,20 @@ def preprocess(self, nb, resources):
         latexdoc_tags = ["code", "error", "table", "equation", "figure", "text"]
         # break up titles
         cells_in_slide = 0
-        header_levels = []
         final_cells = FinalCells(self.header_slide)
+
+        header_levels = []
+        if nb.metadata.toc is not None:
+            base_numbering = nb.metadata.toc.base_numbering
+            if base_numbering is not None:
+                try:
+                    header_levels = list(map(lambda x: int(x), base_numbering.split(".")))
```

Comment:
needs to be formatted with black, see pre-commit tests

Reply:
I don't entirely follow the problem here.  I have fixed the other issues, and have run the pre-commit tests which gives following output:

~~~
(base) sphelps@sphelps-OptiPlex-9010:~/vcs/coding/ipypublish$ pre-commit run --all-files
[WARNING] Unexpected key(s) present on git://github.com/pre-commit/pre-commit-hooks: sha
Check JSON...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
Flake8...................................................................Passed
black....................................................................Passed
RST Linting..............................................................Passed

~~~

Reply:
Well, with the reduced indentation, its passing now anyway, but maybe you don't have `black==19.3b0`?